app/lib/pre_assembly/link_stager.rb

```diff
@@ -4,6 +4,7 @@ module PreAssembly
   # Links are used to stage objects; useful for large files (e.g. media, WARC files)
   class LinkStager
     def self.stage(source, destination)
+      FileUtils.mkdir_p File.dirname(destination)
       FileUtils.ln_s source, destination, force: true
     end
```

Comment:
same comment as the `CopyStager` above -- we need to ensure we create any subfolders as needed